src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -12,6 +12,7 @@ import {
 import type { AssetId } from '@shapeshiftoss/caip'
 import { fromAssetId, thorchainAssetId, thorchainChainId } from '@shapeshiftoss/caip'
 import { SwapperName } from '@shapeshiftoss/swapper'
+import { assetIdToPoolAssetId } from '@shapeshiftoss/swapper/dist/swappers/ThorchainSwapper/utils/poolAssetHelpers/poolAssetHelpers'
```

Comment:
suggestion: Same thing with the swapper package, we might prefer to import the source code instead of the dist file

Reply:
I dont think we ever want to import from packages via `/src` as that'd be treating them as a directory rather than a package which defeats the purpose of having a package.

Better yet, we should move these thorchain helpers into a thorchain package so we arent forced to import deeply from the bowels of swapper